docs/lib-documentation.md

```diff
@@ -262,39 +236,18 @@ Uses [Docker exec](https://docs.docker.com/engine/reference/commandline/exec/) f
 * `exitCode`: The exit code of the command.
 * `logs`: The bytes of the command logs. This isn't a string because Kurtosis can't know what text encoding scheme the container uses.
 
-### generateFiles(Set\<String\> filesToGenerate) -\> Map\<String, [GeneratedFileFilepaths][generatedfilefilepaths]\>
-Generates files inside the enclave data volume, which is mounted on both the container where this code is running and the service container. This allows the current container to write data to files that are immediately available to the service container, as if they shared a filesystem.
-
-**Args**
-
-* `filesToGenerate`: A set of user-defined IDs identifying the files that will be generated.
-
-**Returns**
-
-A map of the file IDs (corresponding to the set passed in as input) mapped to a [GeneratedFileFilepaths][generatedfilefilepaths] object containing the filepaths on a) the testsuite container and b) the service container where the generated file was created.
-
-### loadStaticFiles(Set\<String\> usedStaticFiles) -\> Map\<String, String\>
-Loads static files that have been previously registered with the Kurtosis API via [NetworkContext.registerStaticFiles][networkcontext_registerstaticfiles] into the filespace of the service, so that the service can use it. You'd use this function if, e.g., you had a test database file that you wanted to mount on your service.
-
-**Args**
-
-* `usedStaticFiles`: A set of IDs identifying the static files that should be loaded, where the IDs have been registered with the Kurtosis API in advance via [NetworkContext.registerStaticFiles][networkcontext_registerstaticfiles].
-
-**Returns**
-
-A map of the static file IDs (corresponding to the set passed in as input) mapped to the filepath _inside the service container_ where the static file is now available.
-
+SharedPath
+----------
+Simple structure that holds information about a filepath shared between two containers: this container, and a container running a service in a testnet. The actual object referenced by this path could be anything - a fire, a directory, a symlink, nonexistent, etc.
 
+### getAbsPathOnThisContainer() -\> String
+Gets the absolute path in the container where this code is running
```

Comment:
Thinking on it, might be worth clarifying further (since this object is a bit tricky):
```
For the object in the shared directory represented by this `SharedPath` object, gets the absolute filepath on the container where this code is running.
```